app/javascript/components/journey/Solution.tsx

```diff
@@ -64,14 +64,18 @@ export const Solution = ({
             <GraphicalIcon icon="iteration" />
             {numIterations} {pluralize('iteration', numIterations)}
           </div>
-          <div className="stat">
-            <GraphicalIcon icon="loc" />
-            {numLoc} lines
-          </div>
-          <div className="stat">
-            <GraphicalIcon icon="views" />
-            {numViews} {pluralize('view', numViews)}
-          </div>
+          {numLoc ? (
+            <div className="stat">
+              <GraphicalIcon icon="loc" />
+              {numLoc} lines
```

Comment:
Are there no languages with single line solutions?  Surely there are.  JavaScript even has some.  I think we need `pluralize` here also...

Reply:
Probably. This PR just disables this bit though.